Makefile

```diff
@@ -33,7 +38,12 @@ toitpkg: $(BUILD_DIR)/toitpkg
 TEST_FLAGS ?=
 .PHONY: test
 test: toitpkg $(GO_MOCKS)
-	tedi test -v -cover $(TEST_FLAGS) ./...
+	cmake -E env \
+	    TOITC_PATH=$(TOITC_PATH) \
```

Comment:
Tabs and spaces.

Reply:
Forgot to update this PR after merging...
Probably meant that you reviewed the same change again.
sorry.